css-positioning/relative.css

```diff
@@ -0,0 +1,35 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
```

Comment:
position relative is best to set here, so that you don't have to put it in each selector below